package.json

```diff
@@ -8,23 +8,24 @@
     "buoy": "./bin/buoy.js"
   },
   "scripts": {
-    "watch": "cross-env TARGET=development webpack --config build/config.js --progress --hide-modules --watch",
-    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.dev.config.js --open --hot",
     "build": "npm run build:dist && npm run build:es5",
     "build:dev": "cross-env NODE_ENV=production node build/webpack.dev.config.js",
     "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config build/config.js --progress --hide-modules",
     "build:es5": "rimraf es5 && cross-env NODE_ENV=es5 babel src --out-dir es5",
     "debug-build": "node --inspect --debug-brk build/config.js",
     "debug:test": "./node_modules/.bin/cross-env NODE_ENV=test node --inspect --inspect-brk ./node_modules/jest/bin/jest.js --no-cache --runInBand --verbose",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.dev.config.js --open --hot",
+    "lint": "eslint --ext .js,.vue src",
+    "lint:fix": "eslint --fix --ext .js,.vue src",
```

Comment:
AUTOMATION

Reply:
We said no more `--fix` didn't we?

Reply:
Yes, this has to be run manually. I am leaving it as an option (for cases when we update autofixable rules in the future)

Reply:
I like this.  Gives us a script that will do fixes if we ever decide to do any major changes to the coding style, but won't do anything automatically.